doc/en/components/charts/chart-types-bar.md

```diff
@@ -6,23 +6,118 @@ mentionedTypes: ["XamDataChart"]
 ---
 # $PlatformShort$ Bar Chart
 
-<!-- TODO add introduction about using data-chart -->
+$PlatformShort$ Bar Charts, Bar Graphs, or Horizontal Bar Charts, are among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by horizontal bars or equal width and differing lengths. They are ideal for showing variations in the value of an item over time. Data is represented using a collection of rectangles that extend from the left to right of the chart towards the values of data points. Like a Column Chart, or Column Graph, where bars are displayed vertically (up and down), the bar chart is rendered like the column chart, but with 90 degrees clockwise rotation and a horizontal display (left to right).
 
-## $PlatformShort$ Simple Bar Chart
+There are several common use cases for choosing a Bar Chart: 
```

Comment:
We need a sample after this first paragraph just to show something visual and interactive. We may use the first sample and just have it both here and below. @tosharky

Reply:
https://share.goabstract.com/1e0cf345-060c-49d3-8e04-6fb02aa21e2d?collectionLayerId=c2eae4b8-1622-41e2-b706-867a4fd3a10f&mode=design